src/benchmarks/gc/README.md

```diff
@@ -65,6 +69,34 @@ Finally, run `py . setup` from the same directory as this README.
 This will read information about your system that's relevant to performance analysis (such as cache sizes) and save to `bench/host_info.yaml`.
 It will also install some necessary dependencies on Windows.
 
+_ARM NOTE_: Since build tools do not work on ARM/ARM64, `py . setup` will automatically skip reading and writing the system's information. You will have to get the machine's specs and write `bench/host_info.yaml` independently. This file follows the format shown below (it might vary depending on your machine's NUMA nodes and caches).
+
+```yaml
+hostname:
+n_physical_processors:
+n_logical_processors:
+numa_nodes:
+- numa_node_number:
+  ranges:
+  - lo:
+    hi:
+  cpu_group_number:
+cache_info:
+  l1:
+    n_caches:
+    total_bytes:
+  l2:
+    n_caches:
+    total_bytes:
+  l3:
+    n_caches:
+    total_bytes:
+clock_ghz:
+total_physical_memory_mb:
+```
+
+Most (if not all) of these fields can be retrieved from your machine's _Task Manager_ and under _System_ within _Control Panel_.
```

Comment:
ARM specifics could be moved to a separate doc to keep this one short.
Some brief instruction on how to do this would be nice.
How about Linux ARM devices?

Reply:
Wouldn't this cause redundant information lying around? Most steps are the same with a few deviations here and there. Hence I thought adding it to the README would be more clean.

Reply:
My apologies if I wasn't clear. I meant if we wanted to expand on this (e.g. more detailed description on how to author the yaml file), then we might up those information in a new page to avoid cluttering this one.

Reply:
Ok got it! If we add more information on this on a future PR, we could move it to another file. Maybe one named `docs/host_info_setup.md` or something like that.